02 Algorithm Reference/11 Universes/02 Basic Universe API.html

```diff
@@ -1,10 +1,10 @@
 <meta name="tag" content="universes">
 <meta name="tag" content="custom universes">
 <p><b>Adding Universes</b><br>
-		Universes are added using the <code>AddUniverse()</code> method API. They are a type of data subscription which control what subscriptions are requested; and as such you can create <i>custom</i> universe data types. Depending on what type of universe you are adding there are many helper methods to make it easier. <code>AddUniverse()</code> methods take function filters as parameters, these filters must return an enumerable of Symbol objects.
+		Universes are added using the $[AddUniverse(),M:QuantConnect.Algorithm.QCAlgorithm.AddUniverse] method API. They are a type of data subscription which control what subscriptions are requested; and as such you can create <i>custom</i> universe data types. Depending on what type of universe you are adding there are many helper methods to make it easier. $[AddUniverse(),M:QuantConnect.Algorithm.QCAlgorithm.AddUniverse] methods take function filters as parameters, these filters must return an enumerable of Symbol objects.
 </p>
 <p><b>Universe Settings</b><br>
-			If you do not pass in a full universe object the $[UniverseSettings] property is used to fill in the blanks.  Changing the UniverseSettings algorithm property can be helpful to simplify adding universes. Universes have 5 key properties:
+			If you do not pass in a full universe object the $[UniverseSettings,T:QuantConnect.Data.UniverseSelection.UniverseSettings] property is used to fill in the blanks.  Changing the UniverseSettings algorithm property can be helpful to simplify adding universes. Universes have 5 key properties:
```

Comment:
Same as above: `UniverseSettings` is `P:QuantConnect.Algorithm.QCAlgorithm.UniverseSettings` instead of `T:QuantConnect.Data.UniverseSelection.UniverseSettings`, since we are talking about the helpers that are properties (P).